Dockerfile.ctld

```diff
@@ -0,0 +1,106 @@
+FROM rockylinux:8
```

Comment:
Why do you need to copy the full file? why not to simply extend existing Dockerfile?

Reply:
I believe the current dockerfile is used in the other containers, and we don't want the agent's setup there? Also some python dependencies and pip version were incompatible with our agent. 
Do you mean to use only 1 docker file?